test/unit/puppetlabs/services/jruby/jruby_testutils.clj

```diff
@@ -49,10 +54,14 @@
    :jruby-puppet pool-config
    :certificate-authority {:certificate-status {:client-whitelist []}}})
 
-(defn jruby-puppet-config
+(schema/defn ^:always-validate
+  jruby-puppet-config :- jruby-schemas/JRubyPuppetConfig
   "Create a JRubyPuppet pool config. If `pool-size` is provided then the
```

Comment:
Since we're in here and updating the docstring...can we get rid of this old `pool-size` mention? This function no longer takes that as a parameter.